sdformat_mjcf/src/sdformat_mjcf/mjcf_to_sdformat/converters/geometry.py

```diff
@@ -73,7 +78,12 @@ def mjcf_geom_to_sdf(geom):
         sdf_geometry.set_type(sdf.GeometryType.CYLINDER)
     elif geom.type == "ellipsoid":
         ellipsoid = sdf.Ellipsoid()
-        ellipsoid.set_radii(su.list_to_vec3d(geom.size))
+        if geom.fromto is None:
+            ellipsoid.set_radii(su.list_to_vec3d(geom.size))
+        else:
+            v1 = Vector3d(geom.fromto[0], geom.fromto[1], geom.fromto[2])
+            v2 = Vector3d(geom.fromto[3], geom.fromto[4], geom.fromto[5])
+            ellipsoid.set_radii((v2 - v1).abs())
```

Comment:
The documentation doesn't say that Ellipsoid can use fromto. Should we make 
this an error instead? Otherwise, we need to fix it so that it lines up with 
the behavior of Mujoco, which is to set the radius of the one dimension equal 
to the length of the vector (i.e `(v2 - v1).abs()`) and use the `size` 
attribute for the other radii.